KGHM/index.html

```diff
@@ -1,28 +1,65 @@
-<html>  
-    <head>
-        <meta charset="UTF-8" />
-        <link rel="icon" type="image/svg+xml" href="favicon.svg" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/reset-css@5.0.1/reset.min.css">
-        <link rel="stylesheet" href="style.css">
-        <link rel="stylesheet" href="hangman-pic.css">
-        <title>start KGHM</title>
-    </head>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/reset-css@5.0.1/reset.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>KGHM</title>
+  </head>
 
-    <body>
-        <header>
-            <h1>👅킹맨 한판하실?👅</h1>
-        </header>
+  <body>
+    <header>
+      <h1>행맨 한판하실?</h1>
+    </header>
 
-        <main>
-            <div class="game-wrapper">
-                <section class="life">기회:</section>
-                <section class="hangman">
-                    <div class="hangman_1"></div>
-                </section>
-                <section class="result"></section>
-                <section class="input"></section>
-            </div>
-        </main>
-    </body>
-</html>
\ No newline at end of file
+    <main>
+      <div class="game-wrapper">
+        <section class="life-wrapper">
+          <img src="./public/assets/carrot.png" alt="목숨" />
+          <img src="./public/assets/carrot.png" alt="목숨" />
+          <img src="./public/assets/carrot.png" alt="목숨" />
+          <img src="./public/assets/carrot.png" alt="목숨" />
+          <img src="./public/assets/carrot.png" alt="목숨" />
+          <img src="./public/assets/carrot.png" alt="목숨" />
+          <img src="./public/assets/carrot.png" alt="목숨" />
+          <img src="./public/assets/carrot.png" alt="목숨" />
+          <img src="./public/assets/carrot.png" alt="목숨" />
+          <img src="./public/assets/carrot.png" alt="목숨" />
+        </section>
+        <div class="hangman">
+          <img
+            class="hangman-img"
+            src="./public/assets/1.png"
+            alt="킹맨 사진"
```

Comment:
```suggestion
<img
            class="hangman-img"
            src="./public/assets/1.png"
            alt="킹맨"/>
```

alt 속성은 _ _ 사진 말고 키워드만 적어주는 것이 맞아!-!